Main.html

```diff
@@ -58,8 +60,16 @@ <h3>Личные качества:</h3>
     </div>
   </div>
 
-  <div class="container_2">
-    <h1 id="cont">Контакты</h1>
-    >
+  <div class="container_2" id="cont">
+    <h1>Контакты</h1>
+    <h3>Мои контакты</h3>
+    <ul >
+        <li><img src = "logo/whatsapp.png"><p class="logo_text">+7-951-644-**-**</p></li>
+        <li><img src = "logo/gmail.png"><p class="logo_text">shuric03@gmail.com</p></li>
```

Comment:
Use **a** element for your email address
https://www.rapidtables.com/web/html/mailto.html

Reply:
fixed